netflix/src/redux/films/details.ts

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const detailsSlice = createSlice({
+  name: "details",
+  initialState: {
+    details: undefined,
+  },
+  reducers: {
+    setDetails: (state, film) => {
+      state.details = film.payload;
+    },
+    closeDetails: (state) => {
```

Comment:
`closeDetails()` is same as `setDetails(undefined)`...

Do not afraid to reuse code.

This is not important in such simple case, but if your `setDetails()` has more actions to do - you can avoid code duplication